### ISSUE TITLE
about nuscenes data convert

### ISSUE THREAD

**Issue by AMzhanghan** [hustvl/Senna#15]
The data conversion is very slow, about 6 minutes per frame. Why is it so slow？
![image](https://github.com/user-attachments/assets/d3f34599-89a9-47dd-aab0-c57bb74e78da)

**Comment by rb93dett** [comment]
Yes, the inference speed of LLaVA-1.6-34B is relatively slow, and each sample requires generating multiple QA responses. If you need to speed up data production, you might consider using 4-bit or 8-bit inference [here](https://github.com/hustvl/Senna/blob/main/data_tools/senna_nusc_data_converter.py#L305). If you have sufficient GPU resources, you can also consider splitting the dataset and generating data in parallel.

**Comment by abhigoku10** [comment]
@AMzhanghan  where you able to get the data converted successfully ?? can yo please share the converted data

**Comment by cutefulf** [comment]
the data conversion is so slow,  need almost 100+ days to finish on 8 *3090. who can share the converted data?  many thanks

**Comment by Whale-ice** [comment]
same question

**Comment by casessar** [comment]
> 实在太慢了，大佬们，能提供一下nuscenes v1.0-mini处理好的数据吗？

请教一下佬，有了吗
